### DISCUSSION TITLE
could i use follow-fork in api?

### DISCUSSION THREAD

**Discussion by zdyj3170101136** [bloomberg/memray#536] [Q&A] [answered]

**Comment by pablogsal** [comment]
Please, use the https://github.com/bloomberg/memray/discussions tab for questions

**Comment by pablogsal** [comment, marked as answer]
The answer is yes, you can. Check the API docs here: https://bloomberg.github.io/memray/api.html#memray.Tracker

As you can see you can pass `follow_fork` to the `Tracker` API
